tests/model/model_import_export.test.ts

```diff
@@ -451,7 +451,7 @@ describe("Migrations", () => {
     });
     const data = model.exportData();
     expect(data.version).toEqual(getCurrentVersion());
-    expect(Number(getCurrentVersion())).toBeGreaterThanOrEqual(14.5);
```

Comment:
seems louche 🤔

Reply:
Number() doesn't accept "1.2.3" (or 18.3.1 in our case) as a valid number